src/content/en/fundamentals/instant-and-offline/web-storage/cache-api.md

```diff
@@ -0,0 +1,248 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Learn how to use the Cache API to make your application data available offline
+
+{# wf_updated_on: 2017-09-28 #}
+{# wf_published_on: 2016-09-28 #}
+{# wf_blink_components: Blink>Storage>CacheStorage #}
+
+# Using the Cache API {: .page-title }
+
+{% include "web/_shared/contributors/mscales.html" %}
+
+The Cache API is a system for storing and retrieving network requests and corresponding responses.
+These might be regular requests and responses created in the course of running your application, or
+they could be created solely for the purpose of storing some data in the cache.
+
+The Cache API was created to enable Service Workers to cache network requests so that they can
+provide appropriate responses even while offline. However, the API can also be used as a general
+storage mechanism.
+
+## Where is it available?
+
+The API is currently available in Chrome, Opera and Firefox. Both Edge and Safari have marked the
+API as 'In Development'.
+
+The API is exposed via the global `caches` property, so you can test for the presence of the API
+with a simple feature detection:
+
+    const cacheAvailable = 'caches' in self;
+
+The API can be accessed from a window, iframe, worker, or service worker.
+
+## What can be stored
+
+The caches only store pairs of `Request` and `Response` objects, representing HTTP requests and
+responses, respectively. However, the requests and responses can contain any kind of data that can
+be transferred over HTTP.
+
+Generally speaking, if you are using the API for generic storage, your `Request` objects will not
+need more data than just the URL to be used as a key.
```

Comment:
Sentence reads a bit funny. How about something like:

If you are using the Cache API for generic storage, store the URL for the `Request` object: